Installation issue: python +lzma (--test)

### Steps to reproduce the issue

```console
$ spack spec -I -U python@3.10+lzma ^xz libs=static
Input spec
--------------------------------
 -   python@3.10+lzma
 -       ^xz libs=static

Concretized
--------------------------------
 -   python@3.10.10%gcc@11.3.0+bz2+crypt+ctypes+dbm~debug+libxml2+lzma~nis~optimizations+pic+pyexpat+pythoncmd+readline+shared+sqlite3+ssl~tkinter+uuid+zlib build_system=generic patches=0d98e93,7d40923,f2fd060 arch=linux-ubuntu22.04-zen2
[+]      ^bzip2@1.0.8%gcc@11.3.0~debug~pic+shared build_system=generic arch=linux-ubuntu22.04-zen2
[+]          ^diffutils@3.9%gcc@11.3.0 build_system=autotools arch=linux-ubuntu22.04-zen2
[+]      ^expat@2.5.0%gcc@11.3.0+libbsd build_system=autotools arch=linux-ubuntu22.04-zen2
[+]          ^libbsd@0.11.7%gcc@11.3.0 build_system=autotools arch=linux-ubuntu22.04-zen2
[+]              ^libmd@1.0.4%gcc@11.3.0 build_system=autotools arch=linux-ubuntu22.04-zen2
[+]      ^gdbm@1.23%gcc@11.3.0 build_system=autotools arch=linux-ubuntu22.04-zen2
[+]      ^gettext@0.21.1%gcc@11.3.0+bzip2+curses+git~libunistring+libxml2+tar+xz build_system=autotools arch=linux-ubuntu22.04-zen2
[+]          ^libiconv@1.17%gcc@11.3.0 build_system=autotools libs=shared,static arch=linux-ubuntu22.04-zen2
[+]          ^libxml2@2.10.3%gcc@11.3.0~python build_system=autotools arch=linux-ubuntu22.04-zen2
[+]          ^tar@1.34%gcc@11.3.0 build_system=autotools zip=pigz arch=linux-ubuntu22.04-zen2
[+]              ^pigz@2.7%gcc@11.3.0 build_system=makefile arch=linux-ubuntu22.04-zen2
[+]              ^zstd@1.5.5%gcc@11.3.0+programs build_system=makefile compression=none libs=shared,static arch=linux-ubuntu22.04-zen2
[+]      ^libffi@3.4.4%gcc@11.3.0 build_system=autotools arch=linux-ubuntu22.04-zen2
[+]      ^libxcrypt@4.4.33%gcc@11.3.0~obsolete_api build_system=autotools arch=linux-ubuntu22.04-zen2
[+]          ^perl@5.36.0%gcc@11.3.0+cpanm+open+shared+threads build_system=generic arch=linux-ubuntu22.04-zen2
[+]              ^berkeley-db@18.1.40%gcc@11.3.0+cxx~docs+stl build_system=autotools patches=26090f4,b231fcc arch=linux-ubuntu22.04-zen2
[+]      ^ncurses@6.4%gcc@11.3.0~symlinks+termlib abi=none build_system=autotools arch=linux-ubuntu22.04-zen2
[+]      ^openssl@1.1.1t%gcc@11.3.0~docs~shared build_system=generic certs=mozilla arch=linux-ubuntu22.04-zen2
[+]          ^ca-certificates-mozilla@2023-01-10%gcc@11.3.0 build_system=generic arch=linux-ubuntu22.04-zen2
[+]      ^pkgconf@1.8.0%gcc@11.3.0 build_system=autotools arch=linux-ubuntu22.04-zen2
[+]      ^readline@8.2%gcc@11.3.0 build_system=autotools patches=bbf97f1 arch=linux-ubuntu22.04-zen2
[+]      ^sqlite@3.40.1%gcc@11.3.0+column_metadata+dynamic_extensions+fts~functions+rtree build_system=autotools arch=linux-ubuntu22.04-zen2
[+]      ^util-linux-uuid@2.38.1%gcc@11.3.0 build_system=autotools arch=linux-ubuntu22.04-zen2
[+]      ^xz@5.4.1%gcc@11.3.0~pic build_system=autotools libs=static arch=linux-ubuntu22.04-zen2
[+]      ^zlib@1.2.13%gcc@11.3.0+optimize+pic+shared build_system=makefile arch=linux-ubuntu22.04-zen2

$ spack install -U --test=root python@3.10+lzma ^xz libs=static
```


### Error message

<details><summary>Error message</summary>

<pre>
==> Installing python-3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e
==> No binary for python-3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e found: installing from source
==> Using cached archive: /opt/spack/var/spack/cache/_source-cache/archive/fb/fba64559dde21ebdc953e4565e731573bb61159de8e4d4cedee70fb1196f610d.tgz
==> Applied patch /opt/spack/var/spack/repos/builtin/packages/python/python-3.7.4+-distutils-C++.patch
==> Applied patch /opt/spack/var/spack/repos/builtin/packages/python/python-3.7.4+-distutils-C++-testsuite.patch
==> Applied patch /opt/spack/var/spack/repos/builtin/packages/python/tkinter-3.10.patch
==> Ran patch() for python
==> python: Executing phase: 'configure'
==> python: Executing phase: 'build'
==> python: Executing phase: 'install'
==> Error: ProcessError: Command exited with status 1:
    '/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/python-3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e/bin/python3.10' '-c' 'import lzma'

7 warnings found in build log:
     368     checking for socket in -lsocket... no
     369     checking for --with-libs... no
     370     checking for pkg-config... /opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/pkgconf-1.8.0-jwsn3d4lwnyrybnsf4axeqftbxgsr7xl/bin/pkg-config
     371     checking pkg-config is at least version 0.9.0... yes
     372     checking for --with-system-expat... yes
     373     checking for --with-system-ffi... yes
  >> 374     configure: WARNING: --with(out)-system-ffi is ignored on this platform
     375     checking for --with-system-libmpdec... no
     376     checking for --with-decimal-contextvar... yes
     377     checking for --enable-loadable-sqlite-extensions... yes
     378     checking for --with-tcltk-includes... default
     379     checking for --with-tcltk-libs... default
     380     checking for --with-dbmliborder... gdbm

     ...

     808     /opt/spack/lib/spack/env/gcc/gcc -c -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC   -std=c99 -Wextra -Wno-unused-result -Wno-unused-parameter -Wno-mi
             ssing-field-initializers -Wstrict-prototypes -Werror=implicit-function-declaration -fvisibility=hidden  -I./Include/internal  -I. -I./Include -I/opt/spack/opt/spack/linux-ub
             untu22.04-zen2/gcc-11.3.0/bzip2-1.0.8-sxty3yp5fszl34imibsg7fhydobub2ox/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/expat-2.5.0-sv4xtkrk2aqcou32rofgkdm5a
             zp5rzr7/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gdbm-1.23-wxj4mzly7crmyb7sa3zd62wd3osjjbru/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc
             -11.3.0/gettext-0.21.1-i2ogwq24hd2cyrvfg7bwxf5rsf2ldcxn/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libffi-3.4.4-o76x47hjnp4piij7qyv45pe3owivuzt4/includ
             e -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libxcrypt-4.4.33-5s6ke7m425iw6b7uzywea6ks7idqiks2/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0
             /ncurses-6.4-k6xd5upoe6ohchk77bmch25fxvuqt3jr/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/openssl-1.1.1t-5mb7wr57zf6bmqfnz3vmvqs7pglscxbg/include -I/opt
             /spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/sqlite-3.40
             .1-n224x5z2pycn5ogqlurstmilo5ac5xpz/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/util-linux-uuid-2.38.1-tpzayhwyj4b3xwk4fgjwa6ofi7humtxj/include -I/opt/s
             pack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/xz-5.4.1-t6u26l3fx6ajalkzuyy3zkd4on26j3wy/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/zlib-1.2.13-v3uhv
             noxlp6vk4usuryjac7f7rzcvre6/include  -fPIC -DPy_BUILD_CORE -o Objects/namespaceobject.o Objects/namespaceobject.c
     809     /opt/spack/lib/spack/env/gcc/gcc -c -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC   -std=c99 -Wextra -Wno-unused-result -Wno-unused-parameter -Wno-mi
             ssing-field-initializers -Wstrict-prototypes -Werror=implicit-function-declaration -fvisibility=hidden  -I./Include/internal  -I. -I./Include -I/opt/spack/opt/spack/linux-ub
             untu22.04-zen2/gcc-11.3.0/bzip2-1.0.8-sxty3yp5fszl34imibsg7fhydobub2ox/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/expat-2.5.0-sv4xtkrk2aqcou32rofgkdm5a
             zp5rzr7/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gdbm-1.23-wxj4mzly7crmyb7sa3zd62wd3osjjbru/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc
             -11.3.0/gettext-0.21.1-i2ogwq24hd2cyrvfg7bwxf5rsf2ldcxn/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libffi-3.4.4-o76x47hjnp4piij7qyv45pe3owivuzt4/includ
             e -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libxcrypt-4.4.33-5s6ke7m425iw6b7uzywea6ks7idqiks2/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0
             /ncurses-6.4-k6xd5upoe6ohchk77bmch25fxvuqt3jr/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/openssl-1.1.1t-5mb7wr57zf6bmqfnz3vmvqs7pglscxbg/include -I/opt
             /spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/sqlite-3.40
             .1-n224x5z2pycn5ogqlurstmilo5ac5xpz/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/util-linux-uuid-2.38.1-tpzayhwyj4b3xwk4fgjwa6ofi7humtxj/include -I/opt/s
             pack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/xz-5.4.1-t6u26l3fx6ajalkzuyy3zkd4on26j3wy/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/zlib-1.2.13-v3uhv
             noxlp6vk4usuryjac7f7rzcvre6/include  -fPIC -DPy_BUILD_CORE -o Objects/object.o Objects/object.c
     810     /opt/spack/lib/spack/env/gcc/gcc -c -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC   -std=c99 -Wextra -Wno-unused-result -Wno-unused-parameter -Wno-mi
             ssing-field-initializers -Wstrict-prototypes -Werror=implicit-function-declaration -fvisibility=hidden  -I./Include/internal  -I. -I./Include -I/opt/spack/opt/spack/linux-ub
             untu22.04-zen2/gcc-11.3.0/bzip2-1.0.8-sxty3yp5fszl34imibsg7fhydobub2ox/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/expat-2.5.0-sv4xtkrk2aqcou32rofgkdm5a
             zp5rzr7/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gdbm-1.23-wxj4mzly7crmyb7sa3zd62wd3osjjbru/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc
             -11.3.0/gettext-0.21.1-i2ogwq24hd2cyrvfg7bwxf5rsf2ldcxn/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libffi-3.4.4-o76x47hjnp4piij7qyv45pe3owivuzt4/includ
             e -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libxcrypt-4.4.33-5s6ke7m425iw6b7uzywea6ks7idqiks2/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0
             /ncurses-6.4-k6xd5upoe6ohchk77bmch25fxvuqt3jr/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/openssl-1.1.1t-5mb7wr57zf6bmqfnz3vmvqs7pglscxbg/include -I/opt
             /spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/sqlite-3.40
             .1-n224x5z2pycn5ogqlurstmilo5ac5xpz/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/util-linux-uuid-2.38.1-tpzayhwyj4b3xwk4fgjwa6ofi7humtxj/include -I/opt/s
             pack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/xz-5.4.1-t6u26l3fx6ajalkzuyy3zkd4on26j3wy/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/zlib-1.2.13-v3uhv
             noxlp6vk4usuryjac7f7rzcvre6/include  -fPIC -DPy_BUILD_CORE -o Objects/obmalloc.o Objects/obmalloc.c
     811     /opt/spack/lib/spack/env/gcc/gcc -c -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC   -std=c99 -Wextra -Wno-unused-result -Wno-unused-parameter -Wno-mi
             ssing-field-initializers -Wstrict-prototypes -Werror=implicit-function-declaration -fvisibility=hidden  -I./Include/internal  -I. -I./Include -I/opt/spack/opt/spack/linux-ub
             untu22.04-zen2/gcc-11.3.0/bzip2-1.0.8-sxty3yp5fszl34imibsg7fhydobub2ox/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/expat-2.5.0-sv4xtkrk2aqcou32rofgkdm5a
             zp5rzr7/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gdbm-1.23-wxj4mzly7crmyb7sa3zd62wd3osjjbru/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc
             -11.3.0/gettext-0.21.1-i2ogwq24hd2cyrvfg7bwxf5rsf2ldcxn/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libffi-3.4.4-o76x47hjnp4piij7qyv45pe3owivuzt4/includ
             e -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libxcrypt-4.4.33-5s6ke7m425iw6b7uzywea6ks7idqiks2/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0
             /ncurses-6.4-k6xd5upoe6ohchk77bmch25fxvuqt3jr/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/openssl-1.1.1t-5mb7wr57zf6bmqfnz3vmvqs7pglscxbg/include -I/opt
             /spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/sqlite-3.40
             .1-n224x5z2pycn5ogqlurstmilo5ac5xpz/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/util-linux-uuid-2.38.1-tpzayhwyj4b3xwk4fgjwa6ofi7humtxj/include -I/opt/s
             pack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/xz-5.4.1-t6u26l3fx6ajalkzuyy3zkd4on26j3wy/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/zlib-1.2.13-v3uhv
             noxlp6vk4usuryjac7f7rzcvre6/include  -fPIC -DPy_BUILD_CORE -o Objects/picklebufobject.o Objects/picklebufobject.c
     812     /opt/spack/lib/spack/env/gcc/gcc -c -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC   -std=c99 -Wextra -Wno-unused-result -Wno-unused-parameter -Wno-mi
             ssing-field-initializers -Wstrict-prototypes -Werror=implicit-function-declaration -fvisibility=hidden  -I./Include/internal  -I. -I./Include -I/opt/spack/opt/spack/linux-ub
             untu22.04-zen2/gcc-11.3.0/bzip2-1.0.8-sxty3yp5fszl34imibsg7fhydobub2ox/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/expat-2.5.0-sv4xtkrk2aqcou32rofgkdm5a
             zp5rzr7/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gdbm-1.23-wxj4mzly7crmyb7sa3zd62wd3osjjbru/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc
             -11.3.0/gettext-0.21.1-i2ogwq24hd2cyrvfg7bwxf5rsf2ldcxn/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libffi-3.4.4-o76x47hjnp4piij7qyv45pe3owivuzt4/includ
             e -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libxcrypt-4.4.33-5s6ke7m425iw6b7uzywea6ks7idqiks2/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0
             /ncurses-6.4-k6xd5upoe6ohchk77bmch25fxvuqt3jr/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/openssl-1.1.1t-5mb7wr57zf6bmqfnz3vmvqs7pglscxbg/include -I/opt
             /spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/sqlite-3.40
             .1-n224x5z2pycn5ogqlurstmilo5ac5xpz/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/util-linux-uuid-2.38.1-tpzayhwyj4b3xwk4fgjwa6ofi7humtxj/include -I/opt/s
             pack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/xz-5.4.1-t6u26l3fx6ajalkzuyy3zkd4on26j3wy/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/zlib-1.2.13-v3uhv
             noxlp6vk4usuryjac7f7rzcvre6/include  -fPIC -DPy_BUILD_CORE -o Objects/rangeobject.o Objects/rangeobject.c
     813     /opt/spack/lib/spack/env/gcc/gcc -c -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC   -std=c99 -Wextra -Wno-unused-result -Wno-unused-parameter -Wno-mi
             ssing-field-initializers -Wstrict-prototypes -Werror=implicit-function-declaration -fvisibility=hidden  -I./Include/internal  -I. -I./Include -I/opt/spack/opt/spack/linux-ub
             untu22.04-zen2/gcc-11.3.0/bzip2-1.0.8-sxty3yp5fszl34imibsg7fhydobub2ox/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/expat-2.5.0-sv4xtkrk2aqcou32rofgkdm5a
             zp5rzr7/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gdbm-1.23-wxj4mzly7crmyb7sa3zd62wd3osjjbru/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc
             -11.3.0/gettext-0.21.1-i2ogwq24hd2cyrvfg7bwxf5rsf2ldcxn/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libffi-3.4.4-o76x47hjnp4piij7qyv45pe3owivuzt4/includ
             e -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libxcrypt-4.4.33-5s6ke7m425iw6b7uzywea6ks7idqiks2/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0
             /ncurses-6.4-k6xd5upoe6ohchk77bmch25fxvuqt3jr/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/openssl-1.1.1t-5mb7wr57zf6bmqfnz3vmvqs7pglscxbg/include -I/opt
             /spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/sqlite-3.40
             .1-n224x5z2pycn5ogqlurstmilo5ac5xpz/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/util-linux-uuid-2.38.1-tpzayhwyj4b3xwk4fgjwa6ofi7humtxj/include -I/opt/s
             pack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/xz-5.4.1-t6u26l3fx6ajalkzuyy3zkd4on26j3wy/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/zlib-1.2.13-v3uhv
             noxlp6vk4usuryjac7f7rzcvre6/include  -fPIC -DPy_BUILD_CORE -o Objects/setobject.o Objects/setobject.c
  >> 814     Objects/obmalloc.c:1413:1: warning: 'always_inline' function might not be inlinable [-Wattributes]
     815      1413 | arena_map_get(block *p, int create)
     816           | ^~~~~~~~~~~~~
     817     /opt/spack/lib/spack/env/gcc/gcc -c -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC   -std=c99 -Wextra -Wno-unused-result -Wno-unused-parameter -Wno-mi
             ssing-field-initializers -Wstrict-prototypes -Werror=implicit-function-declaration -fvisibility=hidden  -I./Include/internal  -I. -I./Include -I/opt/spack/opt/spack/linux-ub
             untu22.04-zen2/gcc-11.3.0/bzip2-1.0.8-sxty3yp5fszl34imibsg7fhydobub2ox/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/expat-2.5.0-sv4xtkrk2aqcou32rofgkdm5a
             zp5rzr7/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gdbm-1.23-wxj4mzly7crmyb7sa3zd62wd3osjjbru/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc
             -11.3.0/gettext-0.21.1-i2ogwq24hd2cyrvfg7bwxf5rsf2ldcxn/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libffi-3.4.4-o76x47hjnp4piij7qyv45pe3owivuzt4/includ
             e -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libxcrypt-4.4.33-5s6ke7m425iw6b7uzywea6ks7idqiks2/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0
             /ncurses-6.4-k6xd5upoe6ohchk77bmch25fxvuqt3jr/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/openssl-1.1.1t-5mb7wr57zf6bmqfnz3vmvqs7pglscxbg/include -I/opt
             /spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/sqlite-3.40
             .1-n224x5z2pycn5ogqlurstmilo5ac5xpz/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/util-linux-uuid-2.38.1-tpzayhwyj4b3xwk4fgjwa6ofi7humtxj/include -I/opt/s
             pack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/xz-5.4.1-t6u26l3fx6ajalkzuyy3zkd4on26j3wy/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/zlib-1.2.13-v3uhv
             noxlp6vk4usuryjac7f7rzcvre6/include  -fPIC -DPy_BUILD_CORE -o Objects/sliceobject.o Objects/sliceobject.c
     818     /opt/spack/lib/spack/env/gcc/gcc -c -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC   -std=c99 -Wextra -Wno-unused-result -Wno-unused-parameter -Wno-mi
             ssing-field-initializers -Wstrict-prototypes -Werror=implicit-function-declaration -fvisibility=hidden  -I./Include/internal  -I. -I./Include -I/opt/spack/opt/spack/linux-ub
             untu22.04-zen2/gcc-11.3.0/bzip2-1.0.8-sxty3yp5fszl34imibsg7fhydobub2ox/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/expat-2.5.0-sv4xtkrk2aqcou32rofgkdm5a
             zp5rzr7/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gdbm-1.23-wxj4mzly7crmyb7sa3zd62wd3osjjbru/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc
             -11.3.0/gettext-0.21.1-i2ogwq24hd2cyrvfg7bwxf5rsf2ldcxn/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libffi-3.4.4-o76x47hjnp4piij7qyv45pe3owivuzt4/includ
             e -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libxcrypt-4.4.33-5s6ke7m425iw6b7uzywea6ks7idqiks2/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0
             /ncurses-6.4-k6xd5upoe6ohchk77bmch25fxvuqt3jr/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/openssl-1.1.1t-5mb7wr57zf6bmqfnz3vmvqs7pglscxbg/include -I/opt
             /spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/sqlite-3.40
             .1-n224x5z2pycn5ogqlurstmilo5ac5xpz/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/util-linux-uuid-2.38.1-tpzayhwyj4b3xwk4fgjwa6ofi7humtxj/include -I/opt/s
             pack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/xz-5.4.1-t6u26l3fx6ajalkzuyy3zkd4on26j3wy/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/zlib-1.2.13-v3uhv
             noxlp6vk4usuryjac7f7rzcvre6/include  -fPIC -DPy_BUILD_CORE -o Objects/structseq.o Objects/structseq.c
     819     /opt/spack/lib/spack/env/gcc/gcc -c -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC   -std=c99 -Wextra -Wno-unused-result -Wno-unused-parameter -Wno-mi
             ssing-field-initializers -Wstrict-prototypes -Werror=implicit-function-declaration -fvisibility=hidden  -I./Include/internal  -I. -I./Include -I/opt/spack/opt/spack/linux-ub
             untu22.04-zen2/gcc-11.3.0/bzip2-1.0.8-sxty3yp5fszl34imibsg7fhydobub2ox/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/expat-2.5.0-sv4xtkrk2aqcou32rofgkdm5a
             zp5rzr7/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gdbm-1.23-wxj4mzly7crmyb7sa3zd62wd3osjjbru/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc
             -11.3.0/gettext-0.21.1-i2ogwq24hd2cyrvfg7bwxf5rsf2ldcxn/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libffi-3.4.4-o76x47hjnp4piij7qyv45pe3owivuzt4/includ
             e -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libxcrypt-4.4.33-5s6ke7m425iw6b7uzywea6ks7idqiks2/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0
             /ncurses-6.4-k6xd5upoe6ohchk77bmch25fxvuqt3jr/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/openssl-1.1.1t-5mb7wr57zf6bmqfnz3vmvqs7pglscxbg/include -I/opt
             /spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/sqlite-3.40
             .1-n224x5z2pycn5ogqlurstmilo5ac5xpz/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/util-linux-uuid-2.38.1-tpzayhwyj4b3xwk4fgjwa6ofi7humtxj/include -I/opt/s
             pack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/xz-5.4.1-t6u26l3fx6ajalkzuyy3zkd4on26j3wy/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/zlib-1.2.13-v3uhv
             noxlp6vk4usuryjac7f7rzcvre6/include  -fPIC -DPy_BUILD_CORE -o Objects/tupleobject.o Objects/tupleobject.c
     820     /opt/spack/lib/spack/env/gcc/gcc -c -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC   -std=c99 -Wextra -Wno-unused-result -Wno-unused-parameter -Wno-mi
             ssing-field-initializers -Wstrict-prototypes -Werror=implicit-function-declaration -fvisibility=hidden  -I./Include/internal  -I. -I./Include -I/opt/spack/opt/spack/linux-ub
             untu22.04-zen2/gcc-11.3.0/bzip2-1.0.8-sxty3yp5fszl34imibsg7fhydobub2ox/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/expat-2.5.0-sv4xtkrk2aqcou32rofgkdm5a
             zp5rzr7/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gdbm-1.23-wxj4mzly7crmyb7sa3zd62wd3osjjbru/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc
             -11.3.0/gettext-0.21.1-i2ogwq24hd2cyrvfg7bwxf5rsf2ldcxn/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libffi-3.4.4-o76x47hjnp4piij7qyv45pe3owivuzt4/includ
             e -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libxcrypt-4.4.33-5s6ke7m425iw6b7uzywea6ks7idqiks2/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0
             /ncurses-6.4-k6xd5upoe6ohchk77bmch25fxvuqt3jr/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/openssl-1.1.1t-5mb7wr57zf6bmqfnz3vmvqs7pglscxbg/include -I/opt
             /spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/sqlite-3.40
             .1-n224x5z2pycn5ogqlurstmilo5ac5xpz/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/util-linux-uuid-2.38.1-tpzayhwyj4b3xwk4fgjwa6ofi7humtxj/include -I/opt/s
             pack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/xz-5.4.1-t6u26l3fx6ajalkzuyy3zkd4on26j3wy/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/zlib-1.2.13-v3uhv
             noxlp6vk4usuryjac7f7rzcvre6/include  -fPIC -DPy_BUILD_CORE -o Objects/typeobject.o Objects/typeobject.c

     ...

     1058    /opt/spack/lib/spack/env/gcc/gcc -fPIC -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -std=c99 -Wextra -Wno-unused-result -Wno-unused-parameter -Wno-m
             issing-field-initializers -Wstrict-prototypes -Werror=implicit-function-declaration -fvisibility=hidden -I./Include/internal -I./Include -I. -I/opt/spack/opt/spack/linux-ubu
             ntu22.04-zen2/gcc-11.3.0/bzip2-1.0.8-sxty3yp5fszl34imibsg7fhydobub2ox/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/expat-2.5.0-sv4xtkrk2aqcou32rofgkdm5az
             p5rzr7/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gdbm-1.23-wxj4mzly7crmyb7sa3zd62wd3osjjbru/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-
             11.3.0/gettext-0.21.1-i2ogwq24hd2cyrvfg7bwxf5rsf2ldcxn/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libffi-3.4.4-o76x47hjnp4piij7qyv45pe3owivuzt4/include
              -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libxcrypt-4.4.33-5s6ke7m425iw6b7uzywea6ks7idqiks2/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/
             ncurses-6.4-k6xd5upoe6ohchk77bmch25fxvuqt3jr/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/openssl-1.1.1t-5mb7wr57zf6bmqfnz3vmvqs7pglscxbg/include -I/opt/
             spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/sqlite-3.40.
             1-n224x5z2pycn5ogqlurstmilo5ac5xpz/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/util-linux-uuid-2.38.1-tpzayhwyj4b3xwk4fgjwa6ofi7humtxj/include -I/opt/sp
             ack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/xz-5.4.1-t6u26l3fx6ajalkzuyy3zkd4on26j3wy/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/zlib-1.2.13-v3uhvn
             oxlp6vk4usuryjac7f7rzcvre6/include -I/usr/include/x86_64-linux-gnu -I/usr/local/include -I/tmp/root/spack-stage/spack-stage-python-3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e/s
             pack-src/Include -I/tmp/root/spack-stage/spack-stage-python-3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e/spack-src -c /tmp/root/spack-stage/spack-stage-python-3.10.10-lug3qgn6zb
             fpa3bsmgk5u5xzgvvist6e/spack-src/Modules/socketmodule.c -o build/temp.linux-x86_64-3.10/tmp/root/spack-stage/spack-stage-python-3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e/spac
             k-src/Modules/socketmodule.o
     1059    /opt/spack/lib/spack/env/gcc/gcc -shared -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/bzip2-1.0.8-sxty3yp5fszl34imibsg7fhydobub2ox/lib -L/opt/spack/opt/spack/lin
             ux-ubuntu22.04-zen2/gcc-11.3.0/expat-2.5.0-sv4xtkrk2aqcou32rofgkdm5azp5rzr7/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gdbm-1.23-wxj4mzly7crmyb7sa3zd62wd3o
             sjjbru/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gettext-0.21.1-i2ogwq24hd2cyrvfg7bwxf5rsf2ldcxn/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.
             3.0/libffi-3.4.4-o76x47hjnp4piij7qyv45pe3owivuzt4/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libxcrypt-4.4.33-5s6ke7m425iw6b7uzywea6ks7idqiks2/lib -L/opt/s
             pack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/ncurses-6.4-k6xd5upoe6ohchk77bmch25fxvuqt3jr/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/openssl-1.1.1t-5mb
             7wr57zf6bmqfnz3vmvqs7pglscxbg/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/lib -L/opt/spack/opt/spack/linux-ubu
             ntu22.04-zen2/gcc-11.3.0/sqlite-3.40.1-n224x5z2pycn5ogqlurstmilo5ac5xpz/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/util-linux-uuid-2.38.1-tpzayhwyj4b3xwk4f
             gjwa6ofi7humtxj/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/zlib-1.2.13-v3uhvnoxlp6vk4usuryjac7f7rzcvre6/lib -Wno-unused-result -Wsign-compare -DNDEBUG -g -
             fwrapv -O3 -Wall -fPIC build/temp.linux-x86_64-3.10/tmp/root/spack-stage/spack-stage-python-3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e/spack-src/Modules/_testinternalcapi.o -L
             . -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/bzip2-1.0.8-sxty3yp5fszl34imibsg7fhydobub2ox/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/expat-2.
             5.0-sv4xtkrk2aqcou32rofgkdm5azp5rzr7/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gdbm-1.23-wxj4mzly7crmyb7sa3zd62wd3osjjbru/lib -L/opt/spack/opt/spack/linux
             -ubuntu22.04-zen2/gcc-11.3.0/gettext-0.21.1-i2ogwq24hd2cyrvfg7bwxf5rsf2ldcxn/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libffi-3.4.4-o76x47hjnp4piij7qyv45p
             e3owivuzt4/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libxcrypt-4.4.33-5s6ke7m425iw6b7uzywea6ks7idqiks2/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/g
             cc-11.3.0/ncurses-6.4-k6xd5upoe6ohchk77bmch25fxvuqt3jr/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/openssl-1.1.1t-5mb7wr57zf6bmqfnz3vmvqs7pglscxbg/lib -L/op
             t/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/sqlite-3.40.1-
             n224x5z2pycn5ogqlurstmilo5ac5xpz/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/util-linux-uuid-2.38.1-tpzayhwyj4b3xwk4fgjwa6ofi7humtxj/lib -L/opt/spack/opt/sp
             ack/linux-ubuntu22.04-zen2/gcc-11.3.0/zlib-1.2.13-v3uhvnoxlp6vk4usuryjac7f7rzcvre6/lib -L/usr/lib/x86_64-linux-gnu -L/usr/local/lib -o build/lib.linux-x86_64-3.10/_testinter
             nalcapi.cpython-310-x86_64-linux-gnu.so
     1060    building '_ssl' extension
     1061    /opt/spack/lib/spack/env/gcc/gcc -fPIC -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -std=c99 -Wextra -Wno-unused-result -Wno-unused-parameter -Wno-m
             issing-field-initializers -Wstrict-prototypes -Werror=implicit-function-declaration -fvisibility=hidden -I./Include/internal -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gc
             c-11.3.0/openssl-1.1.1t-5mb7wr57zf6bmqfnz3vmvqs7pglscxbg/include -I./Include -I. -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/bzip2-1.0.8-sxty3yp5fszl34imibsg7fh
             ydobub2ox/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/expat-2.5.0-sv4xtkrk2aqcou32rofgkdm5azp5rzr7/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2
             /gcc-11.3.0/gdbm-1.23-wxj4mzly7crmyb7sa3zd62wd3osjjbru/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gettext-0.21.1-i2ogwq24hd2cyrvfg7bwxf5rsf2ldcxn/inclu
             de -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libffi-3.4.4-o76x47hjnp4piij7qyv45pe3owivuzt4/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/li
             bxcrypt-4.4.33-5s6ke7m425iw6b7uzywea6ks7idqiks2/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/ncurses-6.4-k6xd5upoe6ohchk77bmch25fxvuqt3jr/include -I/opt/
             spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/openssl-1.1.1t-5mb7wr57zf6bmqfnz3vmvqs7pglscxbg/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8
             .2-xbu3istvupcsrd453wmrlkfbcmlr52bq/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/sqlite-3.40.1-n224x5z2pycn5ogqlurstmilo5ac5xpz/include -I/opt/spack/opt/
             spack/linux-ubuntu22.04-zen2/gcc-11.3.0/util-linux-uuid-2.38.1-tpzayhwyj4b3xwk4fgjwa6ofi7humtxj/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/xz-5.4.1-t6u
             26l3fx6ajalkzuyy3zkd4on26j3wy/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/zlib-1.2.13-v3uhvnoxlp6vk4usuryjac7f7rzcvre6/include -I/usr/include/x86_64-lin
             ux-gnu -I/usr/local/include -I/tmp/root/spack-stage/spack-stage-python-3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e/spack-src/Include -I/tmp/root/spack-stage/spack-stage-python-
             3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e/spack-src -c /tmp/root/spack-stage/spack-stage-python-3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e/spack-src/Modules/_ssl.c -o build/tem
             p.linux-x86_64-3.10/tmp/root/spack-stage/spack-stage-python-3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e/spack-src/Modules/_ssl.o
     1062    In file included from /opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/include/readline/readline.h:36,
     1063                     from /tmp/root/spack-stage/spack-stage-python-3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e/spack-src/Modules/readline.c:34:
  >> 1064    /opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/include/readline/rltypedefs.h:35:1: warning: function declaration isn't 
             a prototype [-Wstrict-prototypes]
     1065       35 | typedef int Function () __attribute__((deprecated));
     1066          | ^~~~~~~
  >> 1067    /opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/include/readline/rltypedefs.h:36:1: warning: function declaration isn't 
             a prototype [-Wstrict-prototypes]
     1068       36 | typedef void VFunction () __attribute__((deprecated));
     1069          | ^~~~~~~
  >> 1070    /opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/include/readline/rltypedefs.h:37:1: warning: function declaration isn't 
             a prototype [-Wstrict-prototypes]
     1071       37 | typedef char *CPFunction () __attribute__((deprecated));
     1072          | ^~~~~~~
  >> 1073    /opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/include/readline/rltypedefs.h:38:1: warning: function declaration isn't 
             a prototype [-Wstrict-prototypes]
     1074       38 | typedef char **CPPFunction () __attribute__((deprecated));
     1075          | ^~~~~~~
     1076    building '_hashlib' extension
     1077    /opt/spack/lib/spack/env/gcc/gcc -fPIC -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -std=c99 -Wextra -Wno-unused-result -Wno-unused-parameter -Wno-m
             issing-field-initializers -Wstrict-prototypes -Werror=implicit-function-declaration -fvisibility=hidden -I./Include/internal -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gc
             c-11.3.0/openssl-1.1.1t-5mb7wr57zf6bmqfnz3vmvqs7pglscxbg/include -I./Include -I. -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/bzip2-1.0.8-sxty3yp5fszl34imibsg7fh
             ydobub2ox/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/expat-2.5.0-sv4xtkrk2aqcou32rofgkdm5azp5rzr7/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2
             /gcc-11.3.0/gdbm-1.23-wxj4mzly7crmyb7sa3zd62wd3osjjbru/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gettext-0.21.1-i2ogwq24hd2cyrvfg7bwxf5rsf2ldcxn/inclu
             de -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libffi-3.4.4-o76x47hjnp4piij7qyv45pe3owivuzt4/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/li
             bxcrypt-4.4.33-5s6ke7m425iw6b7uzywea6ks7idqiks2/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/ncurses-6.4-k6xd5upoe6ohchk77bmch25fxvuqt3jr/include -I/opt/
             spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/openssl-1.1.1t-5mb7wr57zf6bmqfnz3vmvqs7pglscxbg/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8
             .2-xbu3istvupcsrd453wmrlkfbcmlr52bq/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/sqlite-3.40.1-n224x5z2pycn5ogqlurstmilo5ac5xpz/include -I/opt/spack/opt/
             spack/linux-ubuntu22.04-zen2/gcc-11.3.0/util-linux-uuid-2.38.1-tpzayhwyj4b3xwk4fgjwa6ofi7humtxj/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/xz-5.4.1-t6u
             26l3fx6ajalkzuyy3zkd4on26j3wy/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/zlib-1.2.13-v3uhvnoxlp6vk4usuryjac7f7rzcvre6/include -I/usr/include/x86_64-lin
             ux-gnu -I/usr/local/include -I/tmp/root/spack-stage/spack-stage-python-3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e/spack-src/Include -I/tmp/root/spack-stage/spack-stage-python-
             3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e/spack-src -c /tmp/root/spack-stage/spack-stage-python-3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e/spack-src/Modules/_hashopenssl.c -o b
             uild/temp.linux-x86_64-3.10/tmp/root/spack-stage/spack-stage-python-3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e/spack-src/Modules/_hashopenssl.o
     1078    In file included from /tmp/root/spack-stage/spack-stage-python-3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e/spack-src/Modules/readline.c:34:
  >> 1079    /opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/include/readline/readline.h:410:1: warning: function declaration isn't a
              prototype [-Wstrict-prototypes]
     1080      410 | extern int rl_message ();
     1081          | ^~~~~~
     1082    /opt/spack/lib/spack/env/gcc/gcc -shared -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/bzip2-1.0.8-sxty3yp5fszl34imibsg7fhydobub2ox/lib -L/opt/spack/opt/spack/lin
             ux-ubuntu22.04-zen2/gcc-11.3.0/expat-2.5.0-sv4xtkrk2aqcou32rofgkdm5azp5rzr7/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gdbm-1.23-wxj4mzly7crmyb7sa3zd62wd3o
             sjjbru/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gettext-0.21.1-i2ogwq24hd2cyrvfg7bwxf5rsf2ldcxn/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.
             3.0/libffi-3.4.4-o76x47hjnp4piij7qyv45pe3owivuzt4/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libxcrypt-4.4.33-5s6ke7m425iw6b7uzywea6ks7idqiks2/lib -L/opt/s
             pack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/ncurses-6.4-k6xd5upoe6ohchk77bmch25fxvuqt3jr/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/openssl-1.1.1t-5mb
             7wr57zf6bmqfnz3vmvqs7pglscxbg/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/lib -L/opt/spack/opt/spack/linux-ubu
             ntu22.04-zen2/gcc-11.3.0/sqlite-3.40.1-n224x5z2pycn5ogqlurstmilo5ac5xpz/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/util-linux-uuid-2.38.1-tpzayhwyj4b3xwk4f
             gjwa6ofi7humtxj/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/zlib-1.2.13-v3uhvnoxlp6vk4usuryjac7f7rzcvre6/lib -Wno-unused-result -Wsign-compare -DNDEBUG -g -
             fwrapv -O3 -Wall -fPIC build/temp.linux-x86_64-3.10/tmp/root/spack-stage/spack-stage-python-3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e/spack-src/Modules/_posixsubprocess.o -L.
              -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/bzip2-1.0.8-sxty3yp5fszl34imibsg7fhydobub2ox/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/expat-2.5
             .0-sv4xtkrk2aqcou32rofgkdm5azp5rzr7/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gdbm-1.23-wxj4mzly7crmyb7sa3zd62wd3osjjbru/lib -L/opt/spack/opt/spack/linux-
             ubuntu22.04-zen2/gcc-11.3.0/gettext-0.21.1-i2ogwq24hd2cyrvfg7bwxf5rsf2ldcxn/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libffi-3.4.4-o76x47hjnp4piij7qyv45pe
             3owivuzt4/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libxcrypt-4.4.33-5s6ke7m425iw6b7uzywea6ks7idqiks2/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gc
             c-11.3.0/ncurses-6.4-k6xd5upoe6ohchk77bmch25fxvuqt3jr/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/openssl-1.1.1t-5mb7wr57zf6bmqfnz3vmvqs7pglscxbg/lib -L/opt
             /spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/sqlite-3.40.1-n
             224x5z2pycn5ogqlurstmilo5ac5xpz/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/util-linux-uuid-2.38.1-tpzayhwyj4b3xwk4fgjwa6ofi7humtxj/lib -L/opt/spack/opt/spa
             ck/linux-ubuntu22.04-zen2/gcc-11.3.0/zlib-1.2.13-v3uhvnoxlp6vk4usuryjac7f7rzcvre6/lib -L/usr/lib/x86_64-linux-gnu -L/usr/local/lib -o build/lib.linux-x86_64-3.10/_posixsubpr
             ocess.cpython-310-x86_64-linux-gnu.so
     1083    /opt/spack/lib/spack/env/gcc/gcc -shared -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/bzip2-1.0.8-sxty3yp5fszl34imibsg7fhydobub2ox/lib -L/opt/spack/opt/spack/lin
             ux-ubuntu22.04-zen2/gcc-11.3.0/expat-2.5.0-sv4xtkrk2aqcou32rofgkdm5azp5rzr7/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gdbm-1.23-wxj4mzly7crmyb7sa3zd62wd3o
             sjjbru/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gettext-0.21.1-i2ogwq24hd2cyrvfg7bwxf5rsf2ldcxn/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.
             3.0/libffi-3.4.4-o76x47hjnp4piij7qyv45pe3owivuzt4/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libxcrypt-4.4.33-5s6ke7m425iw6b7uzywea6ks7idqiks2/lib -L/opt/s
             pack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/ncurses-6.4-k6xd5upoe6ohchk77bmch25fxvuqt3jr/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/openssl-1.1.1t-5mb
             7wr57zf6bmqfnz3vmvqs7pglscxbg/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/lib -L/opt/spack/opt/spack/linux-ubu
             ntu22.04-zen2/gcc-11.3.0/sqlite-3.40.1-n224x5z2pycn5ogqlurstmilo5ac5xpz/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/util-linux-uuid-2.38.1-tpzayhwyj4b3xwk4f
             gjwa6ofi7humtxj/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/zlib-1.2.13-v3uhvnoxlp6vk4usuryjac7f7rzcvre6/lib -Wno-unused-result -Wsign-compare -DNDEBUG -g -
             fwrapv -O3 -Wall -fPIC build/temp.linux-x86_64-3.10/tmp/root/spack-stage/spack-stage-python-3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e/spack-src/Modules/_testmultiphase.o -L. 
             -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/bzip2-1.0.8-sxty3yp5fszl34imibsg7fhydobub2ox/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/expat-2.5.
             0-sv4xtkrk2aqcou32rofgkdm5azp5rzr7/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gdbm-1.23-wxj4mzly7crmyb7sa3zd62wd3osjjbru/lib -L/opt/spack/opt/spack/linux-u
             buntu22.04-zen2/gcc-11.3.0/gettext-0.21.1-i2ogwq24hd2cyrvfg7bwxf5rsf2ldcxn/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libffi-3.4.4-o76x47hjnp4piij7qyv45pe3
             owivuzt4/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libxcrypt-4.4.33-5s6ke7m425iw6b7uzywea6ks7idqiks2/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc
             -11.3.0/ncurses-6.4-k6xd5upoe6ohchk77bmch25fxvuqt3jr/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/openssl-1.1.1t-5mb7wr57zf6bmqfnz3vmvqs7pglscxbg/lib -L/opt/
             spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/sqlite-3.40.1-n2
             24x5z2pycn5ogqlurstmilo5ac5xpz/lib -L/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/util-linux-uuid-2.38.1-tpzayhwyj4b3xwk4fgjwa6ofi7humtxj/lib -L/opt/spack/opt/spac
             k/linux-ubuntu22.04-zen2/gcc-11.3.0/zlib-1.2.13-v3uhvnoxlp6vk4usuryjac7f7rzcvre6/lib -L/usr/lib/x86_64-linux-gnu -L/usr/local/lib -o build/lib.linux-x86_64-3.10/_testmultiph
             ase.cpython-310-x86_64-linux-gnu.so
     1084    building '_sha256' extension
     1085    /opt/spack/lib/spack/env/gcc/gcc -fPIC -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -std=c99 -Wextra -Wno-unused-result -Wno-unused-parameter -Wno-m
             issing-field-initializers -Wstrict-prototypes -Werror=implicit-function-declaration -fvisibility=hidden -I./Include/internal -I./Include -I. -I/opt/spack/opt/spack/linux-ubu
             ntu22.04-zen2/gcc-11.3.0/bzip2-1.0.8-sxty3yp5fszl34imibsg7fhydobub2ox/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/expat-2.5.0-sv4xtkrk2aqcou32rofgkdm5az
             p5rzr7/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/gdbm-1.23-wxj4mzly7crmyb7sa3zd62wd3osjjbru/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-
             11.3.0/gettext-0.21.1-i2ogwq24hd2cyrvfg7bwxf5rsf2ldcxn/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libffi-3.4.4-o76x47hjnp4piij7qyv45pe3owivuzt4/include
              -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/libxcrypt-4.4.33-5s6ke7m425iw6b7uzywea6ks7idqiks2/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/
             ncurses-6.4-k6xd5upoe6ohchk77bmch25fxvuqt3jr/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/openssl-1.1.1t-5mb7wr57zf6bmqfnz3vmvqs7pglscxbg/include -I/opt/
             spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/readline-8.2-xbu3istvupcsrd453wmrlkfbcmlr52bq/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/sqlite-3.40.
             1-n224x5z2pycn5ogqlurstmilo5ac5xpz/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/util-linux-uuid-2.38.1-tpzayhwyj4b3xwk4fgjwa6ofi7humtxj/include -I/opt/sp
             ack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/xz-5.4.1-t6u26l3fx6ajalkzuyy3zkd4on26j3wy/include -I/opt/spack/opt/spack/linux-ubuntu22.04-zen2/gcc-11.3.0/zlib-1.2.13-v3uhvn
             oxlp6vk4usuryjac7f7rzcvre6/include -I/usr/include/x86_64-linux-gnu -I/usr/local/include -I/tmp/root/spack-stage/spack-stage-python-3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e/s
             pack-src/Include -I/tmp/root/spack-stage/spack-stage-python-3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e/spack-src -c /tmp/root/spack-stage/spack-stage-python-3.10.10-lug3qgn6zb
             fpa3bsmgk5u5xzgvvist6e/spack-src/Modules/sha256module.c -o build/temp.linux-x86_64-3.10/tmp/root/spack-stage/spack-stage-python-3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e/spac
             k-src/Modules/sha256module.o -DPy_BUILD_CORE_MODULE

See build log for details:
  /tmp/root/spack-stage/spack-stage-python-3.10.10-lug3qgn6zbfpa3bsmgk5u5xzgvvist6e/spack-build-out.txt

</pre></details>


### Information on your system

* **Spack:** 0.20.0.dev0 (055e5abc93527c97726857a01f67a6f95a0ce075)
* **Python:** 3.10.6
* **Platform:** linux-ubuntu22.04-zen2
* **Concretizer:** clingo

### Additional information

[spack-build-out.txt](https://github.com/spack/spack/files/11337703/spack-build-out.txt)
[spack-build-env.txt](https://github.com/spack/spack/files/11337706/spack-build-env.txt)

@adamjstewart  @pradyunsg  @scheibelp  @skosukhin

### General information

- [X] I have run `spack debug report` and reported the version of Spack/Python/Platform
- [X] I have run `spack maintainers <name-of-the-package>` and **@mentioned** any maintainers
- [X] I have uploaded the build log and environment files
- [X] I have searched the issues of this repo and believe this is not a duplicate

--- a/var/spack/repos/builtin/packages/python/package.py
+++ b/var/spack/repos/builtin/packages/python/package.py
@@ -172,7 +172,7 @@ class Python(Package):
         depends_on("libnsl", when="+nis")
         depends_on("zlib@1.1.3:", when="+zlib")
         depends_on("bzip2", when="+bz2")
-        depends_on("xz", when="+lzma")
+        depends_on("xz libs=shared", when="+lzma")
         depends_on("expat", when="+pyexpat")
         depends_on("libffi", when="+ctypes")
         # https://docs.python.org/3/whatsnew/3.11.html#build-changes

